compose/common.yml

```diff
@@ -116,7 +116,7 @@ services:
       timeout: 1s
       retries: 120
       start_period: 100ms
-    image: ghcr.io/intersectmbo/cardano-db-sync:${CARDANO_DB_SYNC_VERSION:-sancho-4-0-0}
+    image: ghcr.io/intersectmbo/cardano-db-sync:${CARDANO_DB_SYNC_VERSION:-13.2.0.1}
```

Comment:
:memo: I'll need to build this one for ARM, too #1172

Reply:
Done:
* [cardano-db-sync:13.2.0.1](https://github.com/input-output-hk/ogmios-tracker/pkgs/container/ogmios-tracker%2Fcardano-db-sync/197149895?tag=13.2.0.1)